src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -57,6 +58,12 @@ static int        CCSN_Eint_Mode;                  // Mode of obtaining internal
        double     CCSN_CC_Red_DT;                  // reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce
        double     CCSN_MaxRefine_RadFac;           // factor that determines the maximum refinement level based on distance from the box center
        double     CCSN_LB_TimeFac;                 // factor that scales the dt constrained by lightbulb scheme
+       int        CCSN_CC_Rot;                     // mode for rotational profile (0:off, 1:formulated-rotation, 2:IC)
+       double     CCSN_CC_Rot_R0;                  // characteristic radius, R_0, in the formulated rotational profile
+                                                   // ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )
+       double     CCSN_CC_Rot_Omega0;              // central rotational velocity, Omega_0, in the formulated rotational profile
+                                                   // ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )
+       double     CCSN_CC_Rot_Fac;                 // multiplication factor for the rotational profile (only for a tabular rotation)
```

Comment:
Apply the same changes I suggest for `Input__TestProb.CoreCollapse`

Reply:
I applied same comments for this file.